.github/workflows/test-phpunit.yml

```
@@ -110,21 +110,12 @@ jobs:
           DB: ${{ matrix.db-platforms }}
           TERM: xterm-256color
 
-      - name: Run Coveralls
+      - if: matrix.php-versions == '7.4'
+        name: Run Coveralls
         run: |
           composer global require php-coveralls/php-coveralls:^2.4
           php-coveralls --coverage_clover=build/logs/clover.xml -v
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COVERALLS_PARALLEL: true
           COVERALLS_FLAG_NAME: PHP ${{ matrix.php-versions }}
```

Can we try to add DB here? I'm curious if this will give us usable info.
```suggestion
          COVERALLS_FLAG_NAME: PHP ${{ matrix.php-versions }} ${{matrix.db-plaftorms}}
```

---

implemented.